impeller/renderer/backend/vulkan/context_vk.cc

```diff
@@ -128,6 +129,12 @@ void ContextVK::Setup(Settings settings) {
     return;
   }
 
+  submit_message_loop_ = fml::ConcurrentMessageLoop::Create(1u);
+  submit_message_loop_->PostTaskToAllWorkers([]() {
+    // submitKHR is extremely cheap and mostly blocks on an internal fence.
+    fml::RequestAffinity(fml::CpuAffinity::kEfficiency);
```

Comment:
How about just fml::Thread? That has a handy way to name the thread too. You can still post task to it to set it as an efficiency affinity.

Reply:
Oh neat, didn't realize that. makes more sense

Reply:
Done